File: app/src/main/java/com/nan/noisedetector/DetectorApplication.java
package com.nan.noisedetector;

import android.app.Application;

import com.nan.noisedetector.util.PreferenceUtil;

/**
 * @author nan_xu
 * @date 2019/11/22
 */
public class DetectorApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        PreferenceUtil.init(this);
    }
}

File: app/src/main/java/com/nan/noisedetector/DetectorApplication.kt
package com.nan.noisedetector

import android.app.Application
import com.nan.noisedetector.location.LocationManager
import com.nan.noisedetector.util.PreferenceUtil

/**
 * @author nan_xu
 * @date 2019/11/22
 */
class DetectorApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        PreferenceUtil.init(this)
        LocationManager.init(this)
    }
}